ncard_app/templates/test02.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Is this file used in the final product or is it just for testing?

Reply:
just for testing can just ignore it